src/Bundle/Routing/ResourceLoader.php

```diff
@@ -105,17 +107,15 @@ public function supports($resource, $type = null): bool
 
     /**
      * @psalm-suppress InvalidReturnType Symfony docblocks are messing with us
-     *
-     * @return LoaderResolverInterface
      */
-    public function getResolver()
+    public function getResolver(): LoaderResolverInterface
```

Comment:
Since it will crash if `getResolver` will be called before `setResolver`, let's handle it the same as in symfony.

https://github.com/symfony/symfony/blob/bed9c3660c0106049622eead422fcb3921f8dd21/src/Symfony/Component/Config/Loader/Loader.php#L24-L37